devsupApp/src/devsup/__init__.py

```diff
@@ -75,6 +75,15 @@ def _init(iocMain=False):
 device(waveform, INST_IO, pydevsupComIn, "Python Device")
 device(aai, INST_IO, pydevsupComIn, "Python Device")
 device(aao, INST_IO, pydevsupComOut, "Python Device")
+
+device(calcout, INST_IO, pydevsupComOut, "Python Device")
+
+device(int64in, INST_IO, pydevsupComIn, "Python Device")
+device(int64out, INST_IO, pydevsupComOut, "Python Device")
+
+device(lsi, INST_IO, pydevsupComIn, "Python Device")
+device(lso, INST_IO, pydevsupComOut, "Python Device")
```

Comment:
Use of these record types needs needs to be conditional on epics-base version.

- lsi & lso: >= 3.15.0.2
- int64in && int64out: >= 3.16.1

`EPICS_VERSION` and friends are already imported into this file.

Also, if you are willing.  @pheest has pointed out that my usage of `NamedTemporaryFile()` does not work on Windows.  He suggested replacing usage of `NamedTemporaryFile()` with one or more static .dbd files installed alongside this `__init__.py` and loaded using a path computed from `__file__`.  This could be combined, or a separate change.